daycare/models.py

```diff
@@ -39,3 +47,24 @@ def create(email, username, password, name, description, price_per_day, capacity
         new_daycare.save()
 
         return new_daycare
+
+
+class Image(models.Model):
+    name = models.CharField(max_length=255)
+    image = models.CharField(max_length=1000)
```

Comment:
should be `image_url`

Reply:
or even better - url. the Class name already suggests this is representing an image.